alphagov/paas-admin
# Reimplementation of Google analytics with cookie consent preferences management

What
----

A reimplementation of previous tracking - #546 - but only once users consent to it.

It allows users to opt-in or opt-out and not see the banner again for a year

How to review
-------------

- check out branch
- run npm install
- set `this.cookieDomain` to empty string for localhost test in cookie-functions.js
- start the app
- visit http://localhost:3000
- if you haven't set a preference you will see a banner
- save a cookie preference
- check if have cookies stored (chrome: Option + ⌘ + J (on macOS), or Shift + CTRL + J (on Windows/Linux) - Application tab

Who can review
---------------

not @kr8n3r
